app/src/main/java/com/example/concordia_campus_guide/Models/Floor.java

```diff
@@ -70,5 +73,9 @@ public void setAltitude(float altitude) {
     public String getDisplayName(){
         return floorCode;
     }
+
+    public String getCampus() { return campus; }
```

Comment:
This does not have to be in this PR specifically but please when updating/creating new files do not forget to update/create unit tests for the file. In this case, a floor unit test exists so maybe we should update it to test those methods?

Reply:
Ok I updated the tests